.github/workflows/deploy.yml

```diff
@@ -43,29 +43,29 @@ jobs:
       Environment: qa
       Region: us-west-1
       Operation: deploy
-      Version: latest
+      Version: ${{ needs.create-image.outputs.docker_tag }}
     secrets: inherit
 
   prod-via:
-    needs: qa-via
+    needs: [create-image, qa-via]
```

Comment:
Not sure but I think the `prod-*` jobs need to reference `create-image` directly in their `needs` in order to be able to refer to `${{ needs.create-image.outputs.docker_tag }}` below